Prowl.Runtime/Components/Physics/Types/BepuNarrowPhaseCallbacks.cs

```diff
@@ -48,7 +49,7 @@ public unsafe bool ConfigureContactManifold<TManifold>(int workerIndex, Collidab
         var b = CollidableMaterials[pair.B];
         pairMaterial.FrictionCoefficient = a.FrictionCoefficient * b.FrictionCoefficient;
         pairMaterial.MaximumRecoveryVelocity = (float)MathD.Max(a.MaximumRecoveryVelocity, b.MaximumRecoveryVelocity);
-        pairMaterial.SpringSettings = pairMaterial.MaximumRecoveryVelocity == a.MaximumRecoveryVelocity ? a.SpringSettings : b.SpringSettings;
+        pairMaterial.SpringSettings = Math.Abs(pairMaterial.MaximumRecoveryVelocity - a.MaximumRecoveryVelocity) < Application.FloatEqualThreshold ? a.SpringSettings : b.SpringSettings;
```

Comment:
I imagine in the case of BepuPhysics this wouldn't be ideal due to possible performance costs it would come with.
This is very frequently called, and it wouldnt matter much if there are errors due to floating precision here.

Reply:
However, now that I look more closely at the rest of the method, there's a hefty float/double cast and MathD.Max being used. So maybe it doesn't matter as much as I thought.